### ISSUE TITLE
Running atomicapp fails because of wrong cmd for atomic run

### ISSUE THREAD

**Issue by LalatenduMohanty** [projectatomic/atomic#339]
I was trying to run an atomicapp refer https://github.com/projectatomic/adb-atomic-developer-bundle/issues/313#issuecomment-206707112  and failure seems because of  two `-v` i.e. `-v -v` in `docker run -it --rm --privileged -v -v /run:/run -v /:/host --net=host --name helloapache -e NAME=helloapache -e IMAGE=projectatomic/helloapache projectatomic/helloapache run`

See below for details output
```
$ sudo atomic run projectatomic/helloapache
Using default tag: latest
Trying to pull repository docker.io/projectatomic/helloapache ... latest: Pulling from projectatomic/helloapache
1544084fad81: Pull complete 
474c2ee77fa3: Pull complete 
6ee235cf4473: Pull complete 
4816a298548c: Pull complete 
a031cf52cf7b: Pull complete 
baf743b0b5ae: Pull complete 
2e2e83265a20: Pull complete 
5196adc43dab: Pull complete 
5d88c899be7e: Pull complete 
aebfd78101e7: Pull complete 
a80fc0963aae: Pull complete 
a06d4cf90e5e: Pull complete 
3de79d2d4625: Pull complete 
7dd91e3010a6: Pull complete 
b2c7faf089cc: Pull complete 
f96ffbaec8a6: Pull complete 
7ba2613cf60c: Pull complete 
0df252a2fe0b: Pull complete 
82c3b20ede1d: Pull complete 
8e0204d03413: Pull complete 
Digest: sha256:28a041a34604f9fb77cc138794d8bf38e7ab96e70ecd496cfd9428afa477f265
Status: Downloaded newer image for docker.io/projectatomic/helloapache:latest

docker run -it --rm --privileged -v -v /run:/run -v /:/host --net=host --name helloapache -e NAME=helloapache -e IMAGE=projectatomic/helloapache projectatomic/helloapache run

This container uses privileged security switches:

INFO: --net=host 
      Processes in this container can listen to ports (and possibly rawip traffic) on the host's network.

INFO: --privileged 
      This container runs without separation and should be considered the same as root on your system.

For more information on these switches and their security implications, consult the manpage for 'docker run'.

invalid value "-v" for flag -v: -v is not an absolute path
See 'docker run --help'.
```

**Comment by LalatenduMohanty** [comment]
Software versions
```
[vagrant@centos7-adb ~]$ sudo rpm -qa | grep atomic
atomic-1.9-4.gitff44c6a.el7.x86_64

[vagrant@centos7-adb ~]$ cat /etc/centos-release
CentOS Linux release 7.2.1511 (Core) 

[vagrant@centos7-adb ~]$ sudo rpm -qa | grep docker
docker-1.9.1-25.el7.centos.x86_64
python-docker-py-1.7.2-1.el7.noarch
docker-selinux-1.9.1-25.el7.centos.x86_64
docker-forward-journald-1.9.1-25.el7.centos.x86_64
```

**Comment by LalatenduMohanty** [comment]
Reported in atomicapp also https://github.com/projectatomic/atomicapp/issues/673

**Comment by rhatdan** [comment]
@baude PTAL

**Comment by dustymabe** [comment]
We started using ${PWD} in the run labels (https://github.com/projectatomic/atomicapp/commit/f779099) because of https://github.com/projectatomic/atomic/commit/d81e7e651e8eaa55fcbc6d9ddb4b1d7ba9de5ff1 is there any way we can actually pass in the cwd as a volume when we call `sudo atomic run ...`? Running with `sudo -E` is a workaround but not really something we'd like to have to tell our users to do.

**Comment by rhatdan** [comment]
I though the current atomic will grab PWD if it is not set.

```
commit 1f9ec454c7d4fb87b876c2857ac5e2fd1a9965e1
Author: Dan Walsh <dwalsh@redhat.com>
Date:   Thu Mar 10 12:36:20 2016 -0500

    Set PWD environment if not currently set

diff --git a/Atomic/atomic.py b/Atomic/atomic.py
index 9ba96db..646df44 100644
--- a/Atomic/atomic.py
+++ b/Atomic/atomic.py
@@ -613,6 +613,9 @@ class Atomic(object):
         if hasattr(self.args, 'opt3') and self.args.opt3:
             os.environ['OPT3'] = self.args.opt3
 
+        if not hasattr(self.args, 'PWD'):
+            os.environ['PWD'] = os.getcwd()
+
         default_uid = "0"
         with open("/proc/self/loginuid") as f:
             default_uid = f.readline()
```

**Comment by dustymabe** [comment]
we are using `atomic-1.9-4.gitff44c6a.el7.x86_64` from centOS (and I assume RHEL) which doesn't contain that commit I don't believe. What do we need to do to get that resolved?

**Comment by rhatdan** [comment]
Can you wait for atomic-1.10?

**Comment by dustymabe** [comment]
Do we have a timeframe for that?

**Comment by rhatdan** [comment]
atomic-1.10 should be in 7.2.4 release.  We are waiting for a merge from the skopeo/containers in production guys, to make a cut.

**Comment by rhatdan** [comment]
It will be in 7.2.5  Missed the deadline.

**Comment by bexelbie** [comment]
@rhatdan Is there an updated timeframe?  We really need to update docs to deal with this `-E` issue if there is not going to be a fix soon.

**Comment by rhatdan** [comment]
I am planning on cutting atomic-1.10 today.

**Comment by bexelbie** [comment]
<insert lots of positive emoticons>

**Comment by bexelbie** [comment]
@rhatdan can you give us an update?

**Comment by rhatdan** [comment]
atomic-1.10 should be being built today.

**Comment by LalatenduMohanty** [comment]
This is fixed now and atomic-1.10.5-5.el7.x86_64 is available in CentOS and RHEL
